app/helpers/public_helper.rb

```diff
@@ -0,0 +1,2 @@
+module PublicHelper
```

Comment:
Get rid of this file, it's better to create these helpers when they'll be needed.